packages/components/time-field/src/time-field.spec.ts

```diff
@@ -495,7 +511,7 @@ describe('sl-time-field', () => {
     });
 
     it('should have a validation message', () => {
-      expect(el.validationMessage).to.equal('Please enter a time.');
+      expect(el.validationMessage).to.equal('Please fill out this field.');
```

Comment:
Is "Please enter a time" not more user-friendly/clearer than "Please fill out this field"?